TypeCobol/Compiler/Scanner/Token.cs

```diff
@@ -16,7 +16,7 @@ public class Token : Antlr4.Runtime.IToken, IVisitable, IEquatable<Token>
         private int startIndex;
         private int stopIndex;
 
-		/// <summary>Empty constructor for mock.</summary>
+        /// <summary>Empty constructor for mock.</summary>
```

Comment:
Ok but why not changing the next line which also has tabs ?

Reply:
because i see no indent difference when i use Visual Studio editor.